src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -72,6 +67,47 @@ def _post(self, request: Request):
         else:
             return Response("Unsupported version, we only support versions 1 to 3.", 400)
 
+    def _should_use_v3(self, request):
+        version = request.GET.get("version")
```

Comment:
On line 49 this code has a `or "1"` bit presumably to support very old requests.  I guess this is a bug waiting to be triggered.  We should probably either pass version in as an argument here or duplicate the line entirely.  (Personally I'd go with passing it as an argument I think)